WooCommerce/WooCommerce.xcodeproj/project.pbxproj

```diff
@@ -18948,6 +18971,7 @@
 				CLANG_ENABLE_OBJC_WEAK = YES;
 				CODE_SIGN_STYLE = Automatic;
 				INFOPLIST_FILE = WooCommerceScreenshots/Info.plist;
+				IPHONEOS_DEPLOYMENT_TARGET = 17.0;
```

Comment:
@iamgabrielma I think all these should be reverted, since they are probably overriding the common config file.

Reply:
Ooof you're right, sorry for the back and forth. Confirmed that after reverting on https://github.com/woocommerce/woocommerce-ios/pull/16003/commits/d50f5ee8d43ff240ca27aae87fc2ebb417d847ce that we still see the targets individually marked as 17.0 under `General` and builds normally. So it would seem that after the infra changes the only needs are, as you mentioned: 

```
/config/Common.xcconfig
/Modules/Package.swift 
```